packages/design-system/src/style.css

```diff
@@ -36,9 +36,4 @@
   span.line-break-anywhere {
     @apply break-all;
   }
-
-  /* TODO: Might be able to delete after migration */
-  button {
-    @apply h-10 cursor-pointer rounded-xl border-0 px-3 py-[6px] font-sans outline-none active:outline-none;
```

Comment:
not sure I found it, can you point me where those rules are applied now? Or was it just unnecessary somehow?

Reply:
I can see they are replicated in the tab button but what about other buttons of the app?